packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -47,12 +47,11 @@ export const IntlProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
         defaultNS,
         detection: {
           order: [
-            'omsBrowserLanguageDetector',
             'querystring',
             'cookie',
             'localStorage',
             'sessionStorage',
-            'navigator',
```

Comment:
Is removing the navigator correct?

Reply:
the custom detector is a theft of the navigator code, so I thought that we didn't need to duplicate 😉